docs/content/UnderlineNav2.mdx

```diff
@@ -0,0 +1,85 @@
+---
```

Comment:
I think this file should go in `docs/content/drafts/`

Reply:
oops! My bad. Will fix this.,